### DISCUSSION TITLE
rc.46 fails on running migrations

### DISCUSSION THREAD

**Discussion by LudoNieuwenhuizen** [directus/directus#4474] [Archive]
error:
```
code: 'ER_CANT_DROP_FIELD_OR_KEY'
```
When installing a new directus instance, running in AKS, connected to managed mysql (with SSL, which is nowhere documented but does work) and Azure storage account, in case that is relevant. I reverted to rc.44 and stuff works again.

cheers!
Ludo

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/issues/4470#event-4444485325
